controllers/vaultdynamicsecret_controller.go

```diff
@@ -101,9 +101,13 @@ func (r *VaultDynamicSecretReconciler) Reconcile(ctx context.Context, req ctrl.R
 		}
 	}
 
-	var doRolloutRestart bool
+	// forceSync indicates that the controller should perform the secret sync,
```

Comment:
minor nit:  `forceSync` seems to imply that we're doing this when we otherwise wouldn't, now that that is the expected behaviour, I'm wondering if `doSync` is a bit more descriptive?